splinter/driver/lxmldriver.py

```diff
@@ -266,7 +266,7 @@ def _element_is_link(self, element):
         return element.tag == 'a'
 
     def _element_is_control(self, element):
-        return hasattr(element, 'type')
+        return hasattr(element, 'type') or element.tag in ['button', 'input', 'textarea']
```

Comment:
Given that this is now accepting all `<input>`s, I'm not sure there's a need for checking if it has a `type` attribute.